config/config.go

```diff
@@ -26,6 +26,38 @@ func NewConfig() Configuration {
 }
 
 func setConfigDefaults(v *viper.Viper) {
+	v.SetDefault("port", 2424)
+	v.SetDefault("admin_port", 2525)
+	v.SetDefault("log.level", "info")
+	v.SetDefault("backend.type", "memory")
+	v.SetDefault("backend.aerospike.host", "")
+	v.SetDefault("backend.aerospike.port", 0)
+	v.SetDefault("backend.aerospike.namespace", "")
+	v.SetDefault("backend.aerospike.default_ttl_seconds", 0)
+	v.SetDefault("backend.azure.account", "")
+	v.SetDefault("backend.azure.key", "")
+	v.SetDefault("backend.cassandra.hosts", "")
+	v.SetDefault("backend.cassandra.keyspace", "")
+	v.SetDefault("backend.memcache.hosts", []string{""})
```

Comment:
Is there a reason this a slice with one empty string item versus a slice with no items?

Reply:
I saw some post, many months ago (that I cannot find right now), that justified this way as the correct way to initialize default values for a slice if we want environment variable values to be read correctly. In prebid-server this approach has worked, but we could probably run the experiment of setting an empty slice instead. Maybe we don't need that empty string at all. Please let me know if you want me to change it.

Reply:
**Follow-up:** Months ago when we where developing the blacklist and whitelist features for Prebid Server, we noticed how Viper had toruble reading bash arrays into the config. We couldn't quite make it work with a bash array environment variable so we ended up defining the array in the `yaml` file just like we are doing it here. If you want I can remove line 41 above because we'll end up defining the hosts in the `yaml` file anyways. 
```yaml
 29   pbs.yaml: |
 30     gdpr:
 31       non_standard_publishers: [ "STRING1", "STRING2" ] 
base/configmapFiles.yaml
```
In today's research to answer your question, I came across viper's `SetTypeByDefaultValue` function that discusses the case of setting a `[]string{}` via an environment variable. I made some tests but had no success. This is something that will need to be looked into. Do you want me to dig more into this before merging this PR?

Reply:
I tested an empty slice `[]string{}` with no items in PBS and it worked fine.

Reply:
modified